paasta_tools/cli/schemas/deploy_schema.json

```diff
@@ -125,8 +144,5 @@
             "type": "boolean"
         }
     },
-    "required": [
-        "pipeline"
-    ],
```

Comment:
was this meant to be deleted? i think we want to enforce that there's no empty deploy.yaml files

Reply:
I was just here puzzling over why empty deploys started to pass. Yep it was an accident